daq/runner.py

```diff
@@ -195,6 +201,10 @@ def _init_daq_run_id(self):
             output_stream.write(daq_run_id + '\n')
         return daq_run_id
 
+    def _init_device_result_client(self):
+        self._device_result_client = DeviceReportClient(
+            server_port=self.config['device_result_port'])
```

Comment:
best to reference device_result_port only in one spot (similar to other comment)

Reply:
Removed the one in __init__().